bokeh/io.py

```diff
@@ -377,15 +375,12 @@ def save(obj, filename=None, resources=None, title=None, state=None, validate=Tr
     Returns:
         filename (str) : the filename where the HTML file is saved.
 
-    Raises:
-        RuntimeError
-
     '''
     if state is None:
         state = _state
 
     filename, resources, title = _get_save_args(state, filename, resources, title)
-    _save_helper(obj, filename, resources, title, validate)
+    _save_helper(obj, filename, resources, title)
```

Comment:
If they are on public APIs by checking for them and calling `deprecated` explicitly, and then doing whatever fix up is necessary to not propagate the parameter further inside the function.